config/kube.py

```diff
@@ -4,14 +4,20 @@
 import fcntl
 import os
 import shutil
+import sys
 import tempfile
 
+# TODO: Remove this check once we no longer support Python 2
```

Comment:
No need to remove

Reply:
Why not? Once we no longer support Python 2, there is no need to keep this code around. It's better to clean up obsolete code.

Reply:
I am not longer going to support both Python 2 and Python 3 simultaneously. Since this is a library, there is no point.